torch_tvm/compiler.cpp

```diff
@@ -34,15 +37,20 @@ std::vector<DLManagedTensorPtr> set_input(
     //auto tensor = ivalue.toTensor().to(at::kFloat);
     auto tensor = ivalue.toTensor();
     DLManagedTensor* dl_tensor;
-    if (tensor.is_contiguous() &&
-        torch_tvm::utils::isAligned(tensor.data_ptr(),
-          tvm::runtime::kAllocAlignment)) {
+    if tensor.is_contiguous() && (
+        is_training() || torch_tvm::utils::isAligned(tensor.data_ptr(),
```

Comment:
Could you add comment why during training, we require the tensor to be contiguous?

Reply:
we always require tensor to be contiguous (or make it contiguous)

Reply:
OK. Sorry, just comment on why we need special handling for training?

Reply:
sg

Reply:
Previously we were checking not only for contiguity but also for 64B alignment. If either fails then we do copy. The new condition seems to assume that in training data is always aligned to 64B?

Reply:
Or in training we never want to make a copy?

Reply:
we dont' check for alignment here

Reply:
So how are we adding cuda support here? By being able to specify context for devices other than cpu?